dm/loader/lightning.go

```diff
@@ -159,14 +177,15 @@ func (l *LightningLoader) runLightning(ctx context.Context, cfg *lcfg.Config) er
 	if err := l.checkPointList.UpdateStatus(ctx, lightningStatusRunning); err != nil {
 		return err
 	}
+	log2.SetAppLogger(l.logger.Logger)
```

Comment:
dose this cause datarace when multi lighning running at same time? seems it  both tring to replace global logger

and i found we  alreay call `lightning.log.SetAppLogger` when dm-worker start

Reply:
👍